src/common/layouts/Header/Header.jsx

```diff
@@ -29,6 +27,28 @@ const TOOL_LINKS = [
   },
 ];
 
+const ALL_TOOL_LINKS = [
+  {
+    to: '/holdingpen/dashboard',
+    display: 'Holdingpen',
+  },
+  ...UNAUTHORIZED_TOOL_LINKS,
+]
+
+const ONLY_SUPER_USERS_AND_CATALOGERS = Set(['superuser', 'cataloger']);
+
+function isAuthorized(userRoles) {
```

Comment:
should be class function without param, since you can `const { userRoles } = this.props`